src/NexusMods.App.UI/Pages/ModInfo/ModInfoViewModel.cs

```diff
@@ -0,0 +1,102 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.Loadouts.Mods;
+using NexusMods.App.UI.Controls.ModInfo.Error;
+using NexusMods.App.UI.Controls.ModInfo.Loading;
+using NexusMods.App.UI.Controls.ModInfo.ModFiles;
+using NexusMods.App.UI.Pages.ModInfo.Types;
+using NexusMods.App.UI.Resources;
+using NexusMods.App.UI.Windows;
+using NexusMods.App.UI.WorkspaceSystem;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace NexusMods.App.UI.Pages.ModInfo;
+
+public class ModInfoViewModel : APageViewModel<IModInfoViewModel>, IModInfoViewModel
+{
+    [Reactive]
+    public LoadoutId LoadoutId { get; set; }
+    
+    [Reactive]
+    public ModId ModId { get; set; }
+    
+    [Reactive]
+    public CurrentModInfoSection Section { get; set; }
+
+    [Reactive]
+    public IViewModelInterface SectionViewModel { get; set; } = default!;
```

Comment:
I think a puppy died somewhere when you wrote this. Make this nullable or set it to the `DummyLoadingViewModel`.